amy/workshops/tests/test_util.py

```diff
@@ -149,9 +149,9 @@ def test_finding_metadata_on_index(self):
             'address': 'Highway to Heaven 42, Academipolis',
             'latlng': '36.998977, -109.045173',
             'language': 'us',
-            'instructor': 'Hermione Granger, Ron Weasley',
```

Comment:
does this mean that names should also be split with `|` or is this is a test?

Reply:
This is a test for part of AMY that supports reading metatags from YAML - it's a fallback if we can't read them from HTML, and I don't think it's used very often. However, once we read the metatags from YAML, the expected output is that list values (instructors, helpers, or contact) should be joined with "|" as single strings. Originally supported format in YAML is in `self.yaml_content` (lines ca. 80 in this file):

```
instructor: ["Hermione Granger", "Ron Weasley",]
helper: ["Peter Parker", "Tony Stark", "Natasha Romanova",]
contact: ["hermione@granger.co.uk", "rweasley@ministry.gov"]
```